include/GafferArnold/TypeIds.h

```diff
@@ -53,7 +53,7 @@ enum TypeId
 	ArnoldDisplacementTypeId = 110907,
 	ArnoldMeshLightTypeId = 110908,
 
-	LastTypeId = 110949
```

Comment:
I think it's just because I only reserved 1000 for Gaffer in IECore/TypeIds.h in the first place, in a classic "1k ought to be enough for any application" blunder. As you say though, no problem to reshuffle later...